docs/root/configuration/advanced/metadata_configurations.rst

```diff
@@ -0,0 +1,20 @@
+.. _metadata_configurations:
```

Comment:
Can you run this through ChatGPT and ask it to edit for conciseness and clarity.

Reply:
I have let ChatGPT to help me to fix the grammar error but didn't let ChatGPT to revise the entire content.
I will have a try.